core/src/test/scala/org/apache/pekko/persistence/r2dbc/ConnectionFactoryOptionsCustomizerSpec.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
this is not the right source header for new files - see point 5 in https://github.com/apache/pekko/blob/main/CONTRIBUTING.md#pull-request-requirements

Reply:
Fixed